web/packages/design/src/Alert/Alert.tsx

```diff
@@ -145,10 +143,14 @@ export interface AlertProps
   linkColor?: string;
 }
 
-interface ThemedAlertProps extends AlertProps {
+interface ThemedAlertProps extends AlertPropsWithRequiredKind {
   theme: Theme;
 }
 
+type WithRequired<T, K extends keyof T> = T & { [P in K]-?: T[P] };
+
+type AlertPropsWithRequiredKind = WithRequired<AlertProps, 'kind'>;
```

Comment:
I hate this but I couldn't find a better workaround.

Reply:
Maybe it's not that bad actually, we only need it for a couple of internal styled divs. 🤔